.github/workflows/tests.yaml

```diff
@@ -81,19 +81,23 @@ jobs:
         uses: google-github-actions/get-secretmanager-secrets@4d6d3dfd94110800dda8d84109cb6da0f6a5919d # v1.0.1
         with:
           secrets: |-
-            ALLOYDB_URI:${{ secrets.GOOGLE_CLOUD_PROJECT }}/ALLOYDB_URI
+            ALLOYDB_INSTANCE_NAME:${{ secrets.GOOGLE_CLOUD_PROJECT }}/ALLOYDB_INSTANCE_NAME
```

Comment:
is `ALLOYDB_INSTANCE_NAME` the naming we are going forward with over URI?

Reply:
Yes -- that's the correct term. I'm slowly fixing these everywhere.

See https://cloud.google.com/alloydb/docs/reference/rest/v1/projects.locations.clusters.instances/get for confirmation.